blocks/default-output-block/output-types/default.jsx

```diff
@@ -179,47 +238,26 @@ const SampleOutputType = ({
           facebookAdmins={facebookAdmins}
           fallbackImage={fallbackImage}
         />
-
-        <script dangerouslySetInnerHTML={{ __html: ieTest }} />
-        {
-          /** polyfill.io has browser detection and will not load the feature
-           *  if the browser already supports it.
-           */
-        }
-        <script src="https://polyfill.io/v3/polyfill.min.js?features=IntersectionObserver%2CElement.prototype.prepend%2CElement.prototype.remove%2CArray.prototype.find%2CArray.prototype.includes" />
-        {injectStringScriptArray(dangerouslyInjectJS)}
-        <Libs />
+        <link rel="preconnect" href="https://fonts.googleapis.com/" />
+        <link rel="preconnect" href="https://fonts.gstatic.com/" />
```

Comment:
Do we need to explicitly require this? As the call to google fonts eg - https://fonts.googleapis.com/css?family=Roboto+Condensed|Roboto&display=swap

Will return a preconnect header 

```link: <https://fonts.gstatic.com>; rel=preconnect; crossorigin```

Reply:
Oh nice, I didn't know they did that.  Yeah, I'll clean up these.